at-quannguyen/W5/20180718020733_desc_table_user.rb

```diff
@@ -0,0 +1,19 @@
+class DescTableUser < ActiveRecord::Migration[5.1]
+  def change
+    remove_index :users, :email
+    users = User.all
+    users_desc = User.order(id: :desc)
+    users.each_with_index do |elemt, index|
+      obj = users_desc[index]
+      elemt.update_attributes(email: obj.email,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/7b01ff6d5f618841ca8c79e38047e5229d6d21f8/at-quannguyen/W5/20180718020733_desc_table_user.rb#L8"></span>Rails/ActiveRecordAliases: Use <code>update</code> instead of <code>update_attributes</code>.</td>
    </tr>
  </tbody>
</table>